src/css/style.css

```diff
@@ -101,6 +101,71 @@ header:before {
 	border-radius: 50%;
 	background-color: #7E0F11;
 	box-shadow: 0 0 1vh rgba(0,0,0,0.3);
+    z-index: 3;
+}
+
+/* details styles */
+.details-card {
+  position: absolute;
+  top:460px;
+  left:5%;
+  height:1000px;
+  width: 90%;
+  overflow: hidden;
+  background: #FFF;
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.16), 0 2px 10px 0 rgba(0, 0, 0, 0.12);
+  color: #272727;
+  border-radius: 4px;
+  z-index: 2;
+}
+
+.details-header {
+  position: relative;
+  top: 0;
+  left: 0;
+  height:240px;
+  width:100%;
+  background-color: #4a3c31;
+}
+
+.details-name {
+  line-height: 8vw;
+  font-size: 6vw;
+  font-weight: 300;
+  margin:0;
+  padding-top:2vw;
+  padding-bottom:1vw;
+  margin-left:40px;
+  margin-right:40px;
```

Comment:
could delete margin:0; and the left and right margins and just have margin:0 40px;